openapi/openapi.yaml

```diff
@@ -130,6 +154,84 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ZoningDistrict'
+  /zoning-districts/{uuid}/classes:
+    get:
+      summary: Class schemas for the specified zoning district
+      operationId: getZoningDistrictClassesByUuid
+      tags:
+      - Zoning Districts
+      parameters:
+        - name: uuid
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: An array of class schemas for the zoning district
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ZoningDistrictClass'
+  /zoning-district-classes:
+    get:
+      summary: Class schemas for all zoning districts
+      operationId: getAllZoningDistrictClasses
+      tags:
+      - Zoning Districts
+      responses:
+        '200': 
+          description: An array of all zoning district class schemas
+          content:
+            application/json:
+              schema:
+                type: array
+                items: 
+                  $ref: "#/components/schemas/ZoningDistrictClass"
+  /zoning-district-classes/category-colors:
+    get:
+      summary: List of color and class category
+      operationId: getZoningDistrictClassesCategoryColors
+      tags:
+      - Zoning Districts
+    responses:
```

Comment:
The responses parameter needs to be indented with one more tab. I made a branch with the change to resolve any trouble from GitHub code suggestions omitting code validation. 

https://github.com/NYCPlanning/ae-zoning-api/commit/ee989aa5467a0c9e254dad9dfb219ac287e4daf4